python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(cls.gangadir, ignore_errors=True)
```

Comment:
Since it's not trivial to get working I suggest we just remove this function for now. I feel the correct way to fix this would be a basedir which is set on the class with test gangadirs inside it so we can just wipe the basedir each time. Something to sort out later though.